lib/pbench/test/unit/server/query_apis/test_datasets_detail.py

```diff
@@ -85,7 +85,7 @@ def test_query(
                             "authorization": {"user": "unknown", "access": "private"},
                             "run": {
                                 "controller": "dhcp31-187.example.com",
-                                "name": "fio_rhel8_kvm_perf43_preallocfull_nvme_run4_iothread_isolcpus_2020.04.29T12.49.13",
+                                "name": "drb",
```

Comment:
This was a hack to accommodate the common Elasticsearch testing infrastructure, where the fixture denoting login username also identified the dataset name. Otherwise we'd have had to invent extra unnecessary infrastructure to make those connections; it didn't seem worthwhile.